pkg/operator/certrotation/cabundle.go

```diff
@@ -90,6 +87,7 @@ func (c CABundleConfigMap) EnsureConfigMapCABundle(ctx context.Context, signingC
 			reason = fmt.Sprintf("signer update %s", signingCertKeyPairLocation)
 		}
 		c.EventRecorder.Eventf("CABundleUpdateRequired", "%q in %q requires a new cert: %s", c.Name, c.Namespace, reason)
+		updateDetail = fmt.Sprintf("content change: %s", reason)
```

Comment:
mhm, will this remove the reason set inline `72` ?

Reply:
Reworked to make it append (adding a comma if necessary)